games/devtest/mods/testnodes/param2.lua

```diff
@@ -253,7 +253,56 @@ core.register_node("testnodes:color4dir", {
 	groups = { dig_immediate = 3 },
 })
 
+minetest.register_node("testnodes:colorflowingliquid_source", {
```

Comment:
* Should use `core` now
* I would deduplicate these registrations - they share a lot. So maybe introduce a `local function register_liquid_variant` or similar.
* Nitpick: The place where this is inserted also "breaks" the contiguous color4dir "block" in the code.